lighthergm/R/estimate_within_params.R

```diff
@@ -177,11 +179,18 @@ estimate_within_params <-
 
     # Estimate the within-block parameters
     # The default estimation method is "MPLE", but you can select "MLE" if you like.
+    if(is.null(varargs$control)){
+      control <- ergm::control.ergm()
+    } else {
+      control <- varargs$control
+    }
```

Comment:
These could be written in one line, though I'm not sure which is better.
```suggestion
control <- switch(is.null(varargs$control) %>% as.character, "TRUE" = ergm::control.ergm(), "FALSE" = varargs$control)
```

Reply:
@komatsuna4747 
Thanks for the comment. I'd rather not use a `switch` statement for a boolean. I just discovered the `%||%` infix operator. The following one-liner should achieve the same, what do you think?
```R
control <- rlang::`%||%`(varargs$control, ergm::control.ergm())
```

Reply:
@justinian336 
Wow, this is what I have been looking for. I'd prefer `%||%` to the if-else statement in terms of simplicity. But would it be kind to write a comment on what the operator does?

If you use the `%||%` operator, I'd suggest the following:
```r
#' @importFrom rlang %||%

control <- varargs$control %||% ergm::control.ergm()
```

Reply:
@komatsuna4747 thanks for the comment. I made the changes you suggested. Will merge as soon as the checks run successfully.